OpenQA/Isotovideo/Utils.pm

```diff
@@ -12,15 +12,18 @@ use bmwqemu;
 use autotest;
 use Try::Tiny;
 
-our @EXPORT_OK = qw(checkout_git_repo_and_branch checkout_git_refspec
-  handle_generated_assets load_test_schedule);
+our @EXPORT_OK = qw(git_rev_parse checkout_git_repo_and_branch
+  checkout_git_refspec handle_generated_assets load_test_schedule);
+
+sub git_rev_parse ($dirname) {
+    my $addsafe = q{HOME=$(mktemp -d os-autoinst) && git config --global --add safe.directory};
```

Comment:
done

Reply:
`mktemp -d os-autoinst-git.XXXXX --tempdir` will create it in `/tmp`.
But still it needs to be removed afterwards.

I would really still prefer the template, even if we remove it.
It is even more useful in case the directories are *not* removed because of some mistake.
If that happens and we have hundreds of those lying around, we know where it comes from and can react.

Reply:
Ok, `mktemp -d --tmpdir os-autoinst-git.XXXXX` did it. I now included my proposal from https://github.com/os-autoinst/os-autoinst/pull/2104#discussion_r914934993 plus the template plus the proposal from https://github.com/os-autoinst/os-autoinst/pull/2104#discussion_r914954525 with more quotes